terraform/repository_policies.tf

```diff
@@ -573,3 +573,15 @@ module "policy_repository_unique_service_selector" {
     github = github.kubewarden
   }
 }
+
+module "policy_repository_persistentvolumeclaim" {
+  source = "./modules/policy_repository"
+
+  name                   = "persistentvolumeclaim-policy"
```

Comment:
Maybe we will provide other policies related with PVC. What about: "persistentvolumeclaim-storageclass-policy"?

Reply:
I agree with the suggestion

Reply:
`validate-pvc-storageclass-policy`?
or
`pvc-storagelass-policy`? we can then have a series of `pvc-foo-policy`.

Reply:
rebased with `persistentvolumeclaim-storageclass-policy`.